bwt/bwt.go

```diff
@@ -0,0 +1,227 @@
+package bwt
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+const nullChar = "0"
```

Comment:
Is the null char similar to the "$" I've seen appended to the input string in other implementations?